seed/challenges/01-front-end-development-certification/basic-bonfires.json

```diff
@@ -374,7 +374,9 @@
         "assert(confirmEnding(\"He has to give me a new name\", \"name\") === true, 'message: <code>confirmEnding(\"He has to give me a new name\", \"name\")</code> should return true.');",
         "assert(confirmEnding(\"Open sesame\", \"same\") === true, 'message: <code>confirmEnding(\"Open sesame\", \"same\")</code> should return true.');",
         "assert(confirmEnding(\"Open sesame\", \"pen\") === false, 'message: <code>confirmEnding(\"Open sesame\", \"pen\")</code> should return false.');",
+        "assert(confirmEnding(\"Open sesame\", \"game\") === false, 'message: <code>confirmEnding(\"Open sesame\", \"same\")</code> should return true.');",
```

Comment:
Thank you for updating 👍 

Can you amend the message to `should return false` please?

We will be good for a merge after this change :tada: